### DISCUSSION TITLE
Are releases public in a private repository

### DISCUSSION THREAD

**Discussion by ErikKristiansen96** [community/community#126691] [Repositories] [answered]
### Select Topic Area

Question

### Body

This is probably a pretty simpel question, but I can't seem to find an answer to be sure. 
But are releases made from a private repository private or public? Meaning can they be found/downloaded by anyone?

**Comment by davevad93** [comment, marked as answer]
Hi @ErikKristiansen96 ,

As far as I know, a release of a private repo remains private, only the people you have given access to your repo would be able to see it. With GitHub pages it's different, if you have a `PRO` account you can publish your site but it will be public, only `Enterprise` accounts can publish GitHub pages privately.
